conf/SiteDefs.pm

```diff
@@ -26,8 +26,8 @@ sub update_conf {
 
   $SiteDefs::ENSEMBL_COHORT = 'EnsemblGenomes';
   
-  $SiteDefs::SITE_RELEASE_VERSION = 56;
-  $SiteDefs::SITE_RELEASE_DATE    = 'Feb 2023';
+  $SiteDefs::SITE_RELEASE_VERSION = 57;
+  $SiteDefs::SITE_RELEASE_DATE    = 'May 2023';

```

Comment:
Could you please set this to June? Although we're hoping to release in May it's going to be tight, and it's much easier to err on the side of caution than to have to change the date at the last minute!

Reply:
done.